src/components/MetaMaskLink.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const MetaMaskLink = () => (
+  <a
+    href="https://metamask.io/"
+    target="_blank"
+    rel="noopener noreferrer"
+  >
+    Meta Mask
+  </a>
+)
```

Comment:
Linter is neat-picking `11:2  error  Missing semicolon  semi`
https://travis-ci.org/AndreMiras/etheroll/jobs/467164108

Reply:
nice man! will fix